users/validation.py

```diff
@@ -0,0 +1,21 @@
+import re
+
+from django.forms import ValidationError
+
+def Validate_Email(email) :
+    
+    validator = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
+    
+    if not validator.match(email) :
+        raise ValidationError
+    
+    return True
+
+def Validate_Password(password) :
+    
+    validator = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
+    
+    if not validator.match(password) :
+        raise ValidationError
+    
+    return True
```

Comment:
함수명은 이렇게 만드시면 안됩니다. 앞에 대문자가 붙으면 클래스인줄 알고 헷갈릴수가 있어서요.
```suggestion
def validate_email(email) :
    
    validator = re.compile('^[a-zA-Z0-9+-_.]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$')
    
    if not validator.match(email) :
        raise ValidationError
    
    return True

def validate_password(password) :
    
    validator = re.compile("^(?=.*[A-Za-z])(?=.*\d)(?=.*[$@$!%*#?&])[A-Za-z\d$@$!%*#?&]{8,}$")
    
    if not validator.match(password) :
        raise ValidationError
    
    return True
```

Reply:
아하.. 염두하고 수정하겠습니다!